### ISSUE TITLE
Ping: put "no route to host" as status offline

### ISSUE THREAD

**Issue by Kcchouette** [direct-connect/go-dcpp#56]
```
# dcping ping "dchub://78.199.36.164:1411"
2019/03/07 23:57:35 dial tcp 78.199.36.164:1411: connect: no route to host
{"addr":["dchub://78.199.36.164:1411"],"status":"error"}
```

The other hublist put it as offline

**Comment by Kcchouette** [comment]
Same for **no such host**

**Comment by Kcchouette** [comment]
Fixed in v0.8.8
